pyccel/codegen/printing/ccode.py

```diff
@@ -1089,6 +1084,7 @@ def _print_Import(self, expr):
         if source == 'stc/common':
             code = ''
             for t in expr.target:
+                self.add_import(Import(stc_extension_mapping[source], (), ignore_at_print=True))
```

Comment:
Does this line need to be in the loop? Surely it only needs adding once per source not once per target?

Reply:
Good point!